docs/pages/eas-update/github-actions.mdx

```diff
@@ -161,3 +161,41 @@ Your GitHub Action should be set up now. Whenever a developer creates a pull req
 > Some repositories or organizations might need to explicitly enable GitHub Workflows and allow third-party Actions.
 
 </Step>
+
+## Using Bun instead of Yarn
+
+If you're using Bun as your package manager instead of Yarn, you'll need to make some modifications to the provided workflow. Below are the updated steps for both publishing updates on push and previews on pull requests:
+
+Publish updates on push using Bun:
+
+<Step label="1">
+
+Replace the Node setup steps in update.yml:
+
+```- name: Setup Node
+  uses: actions/setup-node@v3
+  with:
+    node-version: 18.x
+    cache: yarn
+```
+with
+```- name: Setup Bun
+  uses: oven-sh/setup-bun@v1
+  with:
+    bun-version: latest
+```
+</Step>
+
+<Step label="2">
+
+Install Dependencies with Bun:
+
+Replace the dependency installation step:
```

Comment:
```suggestion
To install dependencies using Bun, replace the `Install dependencies` step with the following snippet:
```

We can be more clear and add the context here and also following the same patter as step 1 after suggested change.